README.md

```diff
@@ -11,16 +15,16 @@ This is a command-line tool written in Java that performs the following steps:
 # Run the app via command line
 ### Setup
 1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html)
-1. Download [gtfs-validator-v2.0.0_cli.jar](https://github.com/MobilityData/gtfs-validator/releases/download/v2.0.0/gtfs-validator-v2.0.0_cli.jar)
+1. Download the latest snapshot jar of the validator from the artifacts generated by the [`Test Package Document`](https://github.com/MobilityData/gtfs-validator/actions/workflows/test_pack_doc.yml) workflow.
```

Comment:
A few issues here:
1. This link includes PR builds as well - we only want users downloading snapshots built from the master branch. So https://github.com/MobilityData/gtfs-validator/actions/workflows/test_pack_doc.yml?query=branch%3Amaster seems like a better link. But...
2. It's not clear to me where I get the JAR file from on this page. If you can link more directly to that location (preferrably the JAR itself) that would be very helpful - alternately, you'll need to provide more specific instructions here.

Reply:
> This link includes PR builds as well - we only want users downloading snapshots built from the master branch. So https://github.com/MobilityData/gtfs-validator/actions/workflows/test_pack_doc.yml?query=branch%3Amaster seems like a better link. But...

True - I wanted to let the door open for someone who would like to download the jar from a different branch. But since this documentation is dedicated to the `master` branch, it makes sense to use the link you provided. Thanks!

Reply:
> It's not clear to me where I get the JAR file from on this page. If you can link more directly to that location (preferrably the JAR itself) that would be very helpful - alternately, you'll need to provide more specific instructions here.

I'll provide more detailed instructions since it does not appear to have a place to download these jars from a list of jars for example.